nixos/modules/services/logging/logstash.nix

```diff
@@ -23,12 +23,16 @@ let
 
   logstashSettingsYml = pkgs.writeText "logstash.yml" cfg.extraSettings;
 
+  logstashJvmOptionsFile = pkgs.writeText "jvm.options" cfg.extraJvmOptions;
+
   logstashSettingsDir = pkgs.runCommand "logstash-settings" {
+      inherit logstashJvmOptionsFile;
       inherit logstashSettingsYml;
```

Comment:
```suggestion
      inherit logstashJvmOptionsFile logstashSettingsYml;
```

Reply:
I had this originally but then split it up because not-merging independent things into single lines with arbitrary line breaks makes for less merge conflicts / easier cherry-pickability.